### PR TITLE
Core April GA Release

### PR DESCRIPTION
**Pull request by ahsonkhan**

### CODE REVIEW

**Review by antkmsft — COMMENTED** [review]
Looking at the changelog as of now (only bug fixes), it looks like that the next release should be more 1.11.3, rather than 1.12.0?

File: sdk/core/azure-core/CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,6 @@
 # Release History
 
-## 1.12.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.12.0 (2024-04-04)
```
Comment (antkmsft) [inline sdk/core/azure-core/CHANGELOG.md:3]:
```suggestion
## 1.11.3 (2024-04-05)
```

File: sdk/core/azure-core/src/private/package_version.hpp [hunk 2]
```diff
@@ -13,7 +13,7 @@
 #define AZURE_CORE_VERSION_MAJOR 1
 #define AZURE_CORE_VERSION_MINOR 12
 #define AZURE_CORE_VERSION_PATCH 0
```
Comment (antkmsft) [inline sdk/core/azure-core/src/private/package_version.hpp:15]:
```suggestion
#define AZURE_CORE_VERSION_MINOR 11
#define AZURE_CORE_VERSION_PATCH 3
```

**Review by RickWinter — CHANGES_REQUESTED** [review]
We can't disable our coverage gate.

**Review by antkmsft — APPROVED** [review]

**Pull request merged**